apis/pinned-posts/pinned-posts.controller.ts

```diff
@@ -0,0 +1,101 @@
+import * as express from "express";
+import asyncHandler from "express-async-handler";
+import { TypeHelper } from "../../helpers/typeHelper";
+import BitcloutAPI from "../../lib/bitclout/bitclout";
+import JWTAuthMiddleware from "../../middleware/jwt-middleware"
+import { PinnedPostsManager } from "./pinned-posts-manager";
+
+class PinnedPostsController {
+    public router = express.Router();
+
+    private jwtAuthMiddleWare = new JWTAuthMiddleware();
+    private bitclout = new BitcloutAPI();
+    private pinnedPostsManager = new PinnedPostsManager();
+
+    constructor() {
+        this.initializeRoutes();
+    }
+
+    public initializeRoutes() {
+        this.router.post("/pin", this.jwtAuthMiddleWare.protected, asyncHandler(this.pinPost));
+        this.router.post("/unpin", this.jwtAuthMiddleWare.protected, asyncHandler(this.unpinPost));
+        this.router.get("/:publicKey", this.jwtAuthMiddleWare.protected, asyncHandler(this.getPinnedPost));
```

Comment:
Anyone needs to be able to get the pinned post — this shouldn't be protected by middleware